lib/provider/camunda/element-templates/Validator.js

```diff
@@ -405,3 +172,28 @@ function getSchemaVersion(schemaUri) {
 
   return match === null ? undefined : match[0];
 }
+
+/**
+ * Extract only relevant errors of the validation result.
+ *
+ * @param {Array} schemaErrors
+ *
+ * @return {Array}
+ */
+function filteredSchemaErrors(schemaErrors) {
+  return filter(schemaErrors, function(err) {
+
+    // (1) regular errors are customized from the schema
+    if (err.keyword === 'errorMessage') {
```

Comment:
What are "irrelevant" errors that we silently ignore?

Reply:
These are basically errors produced by the JSON Schema which live _alongside_ the primary errors why the validation failed.

As an example, let's have a look [at this example](https://github.com/camunda/element-templates-json-schema/blob/master/test/fixtures/invalid-camunda-in-type.js#L44-L71).

We define rules which have to fail since a given condition is fulfilled. The schema produces one (or more) let's say "primary" errors which indicate this rule. However, the JSON Schema is producing more errors that are not relevant for the user to see.

* One sub-schema (definition) which was expected to succeed is not green --> all the other sub-schemas are also not fulfilled (e.g. the template itself is no array)
* `oneOf` is not fulfilled for the rules
* The `if` clause is not fulfilled 
* ....

This brings us to the assumption: all relevant errors rather have a custom error message defined by us (which is the case currently) or are simple data type errors.

Reply:
I hope that clarifies it a bit, we can add more comments to make it clearer if we want to.

Reply:
Makes sense. Would be beneficial to have a summary of what you said in the code I guess. I'd otherwise struggle to understand what this is about, next time I check.

Reply:
Will add 👍